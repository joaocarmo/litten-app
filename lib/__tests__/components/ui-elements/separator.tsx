import TestRenderer from 'react-test-renderer'
import ThemeProvider from '@components/theme/provider'
import { UISeparator } from '@ui-elements'

describe('Snapshot test for the "UISeparator" component', () => {
  it('renders correctly', () => {
    const element = TestRenderer.create(
      <ThemeProvider>
        <UISeparator />
      </ThemeProvider>,
    ).toJSON()
    expect(element).toMatchSnapshot()
  })
})
