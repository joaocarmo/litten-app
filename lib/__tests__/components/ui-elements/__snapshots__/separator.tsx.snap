// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for the "UISeparator" component renders correctly 1`] = `
<View
  style={
    Array [
      Array [
        Object {
          "marginBottom": 20,
          "marginTop": 20,
        },
        Object {
          "backgroundColor": "#c4c4c4",
          "height": 0.5,
        },
      ],
      undefined,
    ]
  }
/>
`;
